elastic/kibana
# [eem] disable entity discovery on plugin startup

Disable entity discovery on plugin startup and remove some related files.

We'll need a follow up to cleanup endpoints.

### Testing

- enable entity discovery from `main` with `PUT kbn:/internal/entities/managed/enablement`, preferably in a cluster with data so you can verify that the `.entities*` indices are also deleted
- checkout that branch and start kibana
- verify all v1 builtin definitions with their transforms/templates/pipelines are deleted. `.entities` indices for these definitions should also be gone
